luigi/worker.py

```diff
@@ -342,21 +343,65 @@ def respond(self, response):
         self._scheduler.add_scheduler_message_response(self._task_id, self._message_id, response)
 
 
+class SyncResult(object):
+    """
+    Synchronous implementation of ``multiprocessing.pool.AsyncResult`` that immediately calls *func*
+    with *args* and *kwargs*. Its methods :py:meth:`get`, :py:meth:`wait`, :py:meth:`ready` and
+    :py:meth:`successful` work in a similar fashion, depending on the result of the function call.
+    """
+
+    def __init__(self, func, args=None, kwargs=None):
+        super(SyncResult, self).__init__()
```

Comment:
luigi isn't officially supporting py 2.7 anymore so this could just be `super().__init__()`, but totally optional